tests/auxiliary/user-account/test_contract.py

```diff
@@ -347,6 +355,7 @@ def test_no_call_if_not_enough_gas(vm):
     assert computation.gas_meter.gas_remaining > 0
 
 
+@pytest.mark.xfail(reason="obsolete gas payment mechanism, #234, and #281", strict=True)
```

Comment:
I'm fine with leaving these this way *if* you will 1) create an issue to track the cleanup, and 2) you expect to address this in the near future.

Otherwise I'd be worried about us forgetting about these, them no longer actually failing, but the test suite continuing to pass.